ansible/roles/riakcs/tasks/admin_user.yml

```diff
@@ -12,7 +12,7 @@
 
 # Note this can't be notify because we need it to restart immediately
 - name: Restart RiakCS
-  command: riak-cs restart
+  service: name=riak-cs state=started enabled=yes
```

Comment:
Why are there two tasks in the file called `Restart RiakCS`? (see also https://github.com/dimagi/commcarehq-ansible/blob/fix-riak-start/ansible/roles/riakcs/tasks/admin_user.yml#L67)